src/components/GameField/GameField.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useMemo, useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { ChangeDirectionActions } from '../../store/types';
+import { directionActionCreator } from './../../store/actions';
+import './GameField.css'
+
+const GameField = () => {
```

Comment:
I think GameField component should accept props like 'rows' and 'cols' to allow creating fields of various sizes